inc/MediaResponse.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Media Response object.
+ *
+ * @package asset-manager-framework
+ */
+
+declare( strict_types=1 );
+
+namespace AssetManagerFramework;
+
+class MediaResponse {
+
+	/**
+	 * The reponse media list.
+	 *
+	 * @var MediaList
+	 */
+	private $media_list;
+
+	public function __construct( MediaList $media_list, int $total, int $per_page ) {
+		$this->media_list = $media_list;
```

Comment:
I suggest providing defaults for this to allow for easier usage in failure/empty case:

```suggestion
	public function __construct( MediaList $media_list = null, int $total = 0, int $per_page = 40 ) {
		$this->media_list = $media_list ?? new MediaList();
```